conanfile.py

```diff
@@ -43,7 +43,8 @@ class QtConan(ConanFile):
     _submodules = _getsubmodules()
 
     generators = "pkg_config"
-    name = "qt"
+    name = "Qt"
+    version = "5.12.5"
```

Comment:
Due to how we build in the team city infrastructure, I need to include the version in the recipe because I'm not able to define it via the command line args.  i.e. `Qt/5.12.5@black-coral/release`